shared/common-adapters/standard-screen.js.flow

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+import {Component} from 'react'
+
+export type Props = {
+children?: ?(React$Element<*> | Array<React$Element<*>>),
```

Comment:
Should this be indented? Also is there a type that includes the array possibility? Something like `React.PropTypes.node`?